gtsam/navigation/CombinedImuFactor.cpp

```diff
@@ -72,6 +72,17 @@ bool PreintegratedCombinedMeasurements::equals(
 
 //------------------------------------------------------------------------------
 void PreintegratedCombinedMeasurements::resetIntegration() {
+  // Set the initial bias covariance to
+  // the estimated covariance from the last step.
+  p().biasAccOmegaInt = preintMeasCov_.block<6, 6>(9, 9);
```

Comment:
I don't think this is correct. The propagated noise is not the same as the uncertainty on the bias at t1. Ideally we would use the marginal covariance on the bias but we can't expect the information to be available. So I think when resetting without any arguments the biasAccOmegaInt should be left alone.

Reply:
When propagating the noise, shouldn't this be the new (propagated) biasAccOmegaInit after the last preintegration step?
Please correct me if I am wrong, but my understanding was that we should automatically be picking the updated/propagated biasAccOmegaInit from the last time step (t_k, not t1) and setting it for the next step so we don't use the original biasAccOmegaInit set at the very beginning.

Reply:
Indeed, I think this might be incorrect. @varunagrawal , we should talk.

Reply:
Hi @varunagrawal, would it be possible to report back on the resolution of this to improve understanding? It might also be helpful to update the `ImuFactor.pdf` on the role of `\eps_init` in the model (eq 15, 16) --- is this just a zero-mean Gaussian prior on the bias when the IMU turned on? In which case, what is an example use case / motivation for wanting to set a new `biasAccOmegaInt` upon `resetIntegration`?

Reply:
also, it is probably worth renaming `biasAccOmegaInt` --> `biasAccOmegaInit` to be more consistent and to avoid potential confusion with the integration noise.

Reply:
@plusk01 yes it is a gaussian for when we start the IMU. The idea for setting a new `biasAccOmegaInit` is that when we reset the integration between $t_k$ and $t_{k+1}$, we are still using the old `biasAccOmegaInit` rather than a new covariance for $t_{k+1}$. As @raabuchanan mentioned, we could compute this new value from the marginal on the bias.

Regarding the naming of `biasAccOmegaInt` to `biasAccOmegaInit`, I'll need @dellaert's approval for that.

Reply:
@plusk01 `biasAccOmegaInt` is a typo and should be `biasAccOmegaInit`. It is the same variable as Q_init in `ImuFactor.pdf` except you pass a single value to the factor and it is converted to a diagonal matrix.